aiogram/api/client/session/base.py

```diff
@@ -30,6 +33,9 @@ def __init__(
         self.api = api
         self.json_loads = json_loads
         self.json_dumps = json_dumps
+        self.proxy = proxy
+
+        self.cfg: types.SimpleNamespace = types.SimpleNamespace()
```

Comment:
I don't like short name for this attribute. What do you think about something like `properties` or `config`?

And also i have question why do you use NS instead of adding separated attributes?

Reply:
Yep, `config` would be nicer.
I guess, I didn't want to mess session's namespace and keep "configs" as simple namespace.

Reply:
By the way, why is it named config and placed in BaseSession? It's only my opinion, but private attributes like `_connector_type` and `_connector_params` in AiohttpSession would be better

Reply:
It's inner configuration that designed to be in one namespace as I commented above.

Reply:
:disappointed: Since the `SimpleNamespace` was not met as I wanted, I'll just set new attributes starting with underscore. Thanks for the opinions and arguments on implementation disadvantages.